plugins/org.jkiss.dbeaver.ui.dashboard/src/org/jkiss/dbeaver/ui/dashboard/view/DashboardUpdater.java

```diff
@@ -379,26 +380,32 @@ private DashboardDataset transposeDataset(DashboardDataset dataset) {
 
     public List<DashboardContainer> getDashboardsToUpdate() {
         List<DashboardContainer> dashboards = new ArrayList<>();
+        int activeViewDashboards = 0;
         for (IWorkbenchWindow window : PlatformUI.getWorkbench().getWorkbenchWindows()) {
             for (IWorkbenchPage page : window.getPages()) {
                 for (IViewReference view : page.getViewReferences()) {
                     if (view.getId().equalsIgnoreCase(DashboardView.VIEW_ID)) {
                         IWorkbenchPart part = view.getPart(false);
                         if (part instanceof DashboardView) {
-                            getViewDashboards((DashboardView) part, dashboards);
+                            activeViewDashboards += getViewDashboards((DashboardView) part, dashboards);
                         }
                     }
                 }
             }
         }
+        if (0 == activeViewDashboards) {
```

Comment:
This doesn't apply to Java because there are no implicit conversions from `int` to `boolean`.

```suggestion
        if (activeViewDashboards == 0) {
```